config/routes.rb

```diff
@@ -8,4 +8,9 @@
   resources :quizzes
   resources :response
   get 'quiz', to: 'quizzes#quiz', as: :take_quiz
+
+  get 'profile/:id'=> 'profile#show', :as => 'show'
```

Comment:
Should probably rename this and the next one to "show_profile" and "edit_profile" to prevent ambiguity and future issues.

Reply:
Don't forget to update usages of these path aliases in your profile_controller.rb!